docs/quickstart.md

```diff
@@ -43,47 +42,143 @@ Install the Fleet Helm charts (there's two because we separate out CRDs for ulti
     fleet/fleet`}
 </CodeBlock>
 
+To verify installation, run:
+
+`kubectl get pods -n cattle-fleet-system`
+
 ## Add a Git Repo to Watch
 
-Change `spec.repo` to your git repo of choice.  Kubernetes manifest files that should
-be deployed should be in `/manifests` in your repo.
+Specify the Git repositories containing your deployment manifests or Helm charts. For hello world, example you need:
+* deployment.yaml for defining workload
+* fleet.yaml for how should fleet control  the deployment
+* Gitrepo.yaml for where to find your git repo, which branch and sub-path to monitor.
+  * (optionally) You can also add credentials
 
-```bash
-cat > example.yaml << "EOF"
+Structure your repository like this:
+
+![Screenshot displaying the file directory](../static/img/file-structure-sample-ss.png)
```

Comment:
I would not suggest to have the `gitrepo.yaml` in the repository being deployed. 

The reason why is because the GitRepo can be considered as just another resource by Fleet and can add confusion.

For example... lets say you have a git repository with some YAML fields and that one of them is a GitRepo. 

Fleet will deploy that GitRepo, which means it will create another GitRepo (you are nesting GitRepos).
You might want to do that, but I don't think that's the case in this example.

I would take the GitRepo out of the git repository here to avoid confusions and unexpected behaviour.

Reply:
+1

It was confusing to me as well the first time I deploy an application using Fleet
I think I would also add a warning about this bad approach